migrations/v4.js

```diff
@@ -0,0 +1,58 @@
+import { describe, whereContent, whereFromPlugin, mutateContent, checkContent, updatePlugin } from 'adapt-migrations';
+import _ from 'lodash';
+
+describe('Tutor - v2.1.0 to v4.0.0', async () => {
+  let course, courseTutorGlobals, components;
+  whereFromPlugin('Tutor - from v2.1.0', { name: 'adapt-contrib-tutor', version: '<4.0.0' });
+  whereContent('Tutor - where ?', async (content) => {
+    // Check if Tutor is installed?? Configuration won't exist at this point.
+    // return true;
+
+    // Check if any components exist and set components variable
+    components = content.filter(({ _component }) => _component);
+    return components.length;
+  });
```

Comment:
@oliverfoster Tutor <4.0.0 won't have any configuration in the JSON. I'm not sure what to use for the `whereContent` here. Should I check that Tutor has been installed? I'm currently just using it to set the components variable since I update all components later in this `describe()`.

Reply:
The where content and whereFromPlugin are separate so that you don't have to specify either. If you have no whereContent it is fine to rely on whereFromPlugin.